src/python/tools/db2graph/marius_db2graph.py

```diff
@@ -0,0 +1,709 @@
+            raise ValueError("{} does not exist".format(str(query_filepath)))
+
+        file = open(query_filepath, 'r')
+        read_lines = file.readlines()
+        for i in range(len(read_lines)):
+            # Removing the last '\n' character
+            if (read_lines[i][-1] == '\n'):
+                read_lines[i] = read_lines[i][:-1]
+            
+            # Adding the line to rel_list if even else its a query
+            if (i % 2 == 0):
+                edge_entity_feature_values_rel_list.append(read_lines[i])
+            else:
+                edge_entity_feature_values_sql_queries.append(read_lines[i])
+    else:
+        logging.error("ERROR: edges_entity_feature_values_queries is not defined")
+        exit(1)
+
+    return db_server, db_name, db_user, db_password, db_host, generate_uuid, entity_node_sql_queries, edge_entity_entity_sql_queries,\
+     edge_entity_entity_rel_list, edge_entity_feature_values_sql_queries, edge_entity_feature_values_rel_list
+
+def connect_to_db(db_server, db_name, db_user, db_password, db_host):
+    """
+    Function takes db_server and db_name as the input. Tries to connect to the database and returns an object
+    which can be used to execute queries.
+    Assumption: default user: root, host: 127.0.0.1 and password:"". You will need to change code if otherwise
+
+    :param db_server: The name of the backend database application used for accessing data
+    :param db_name: The name of the database where the data resides
+    :param db_user: The user name used to access the database
+    :param db_password: The password used to access the database
+    :param db_host: The hostname of the database
+    :return cnx: cursor object that can be used to execute the database queries
+    """
+    if db_server == 'maria-db' or db_server == 'my-sql':
+        try:
+            cnx = mysql.connector.connect(user=db_user,
+                                        password=db_password,  # change password to your own
+                                        host=db_host,
+                                        database=db_name)
+            # cursor = cnx.cursor(name="my_cursor") # Make cursor only when you need it
+        except mysql.connector.Error as err:
+            if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+                logging.error(f"Incorrect user name or password\n{err}")
+            elif err.errno == errorcode.ER_BAD_DB_ERROR:
+                logging.error(f"Non-existing database\n{err}")
+            else:
+                logging.error(err)
+
+    elif db_server == 'postgre-sql':
+        try:
+            cnx = psycopg2.connect(user=db_user,
+                                 password=db_password,  # change password to your own
+                                 host=db_host,
+                                 database=db_name)
+            # Note: It is important to have cursor name in psycopg else all the data will be transfered 
```

Comment:
Is this comment and the correspond commented code needed?

Reply:
Not needed removed.